templates/wip-card.html

```diff
@@ -0,0 +1,25 @@
+<div class="column">
```

Comment:
Would be nice if we could set a standard size for these instead of having them scale based on contents. Right now all three on the preview are different.